src/modules/map/components/attributionInfo/AttributionInfo.js

```diff
@@ -0,0 +1,77 @@
+import { html, nothing } from 'lit-html'; 
+import { BaElement } from '../../../BaElement';
+import { $injector } from '../../../../injection';
+import css from './attributionInfo.css';
+
+/**
+ * a class for displaying the attribution of the basemap
+ * @class
+ * @author bakir_en 
+ */
+export class AttributionInfo extends BaElement {
+
+	constructor() {
+		super();
+		const { TranslationService, GeoResourceService } = $injector.inject('TranslationService', 'GeoResourceService');
+		this._translationService = TranslationService;
+		this._georesourceService = GeoResourceService;
+	} 
+
+
+	/**
+     * @override 
+     */
+	createView() {
+		const translate = (key) => this._translationService.translate(key);
+		const { active, zoom } = this._state;
+
+		const geoResource = active[0] ?  this._georesourceService.byId(active[0].id) : null;
```

Comment:
Sorry, did not see this the last time:
In contrast to the base layer info (where we take the current base layer only), here we have to display the attribution for **all active AND visible layers**